modules/openapi-generator/src/main/resources/python/python-experimental/model_templates/methods_setattr_getattr_composed.mustache

```diff
@@ -20,28 +20,22 @@
                     )
             return None
 
-        path_to_item = []
-        if self._path_to_item:
-            path_to_item.extend(self._path_to_item)
-        path_to_item.append(name)
         raise ApiAttributeError(
             "{0} has no attribute '{1}'".format(
                 type(self).__name__, name),
-            path_to_item
+            [e for e in [self._path_to_item, name] if e]
```

Comment:
How about just using name here? We know that name was not found.
- this same comment applies on line 70 too

Reply:
i just compacted the previous functionality to find `path_to_item`, if you think just `name` is enough instead of the old path code, i'm fine with that